jest.config.js

```diff
@@ -3,6 +3,9 @@ module.exports = {
   testRegex: '/test/.*\\.test\\.tsx$',
   modulePathIgnorePatterns: ['<rootDir>/examples/'],
   setupFilesAfterEnv: ['<rootDir>/jest-setup.ts'],
+  moduleNameMapper: {
+    swr: '<rootDir>/dist/index.js'
```

Comment:
guess we should use source instead of dist for testing?

Reply:
We have to do this, because the source code of infinite is using `import useSWR from 'swr'`.